src/Potree.js

```diff
@@ -663,6 +663,11 @@ Potree.updateVisibility = function(pointclouds, camera, renderer){
 		}
 
 		if (node.isTreeNode()) {
+			if (!pointcloud.initialLoadFinished) {
+				pointcloud.initialLoadFinished = true;
+				pointcloud.dispatchEvent({ type: 'pointcloud_ready' });
```

Comment:
This string, shouldn't it be a constant somewhere?

Reply:
This could be a constant but current, master version of Potree does not use constants for event names and I didn't want to change too much code across this package. The reason is that some people are working on refactoring Potree (e.g. detach UI from core functionality, make it more NPM friendly) on `develop` branch and I would like to transition to that version someday.

Reply:
Nice!